src/include/server.h

```diff
@@ -188,10 +188,11 @@ struct server {
 	struct server_qs {
 		int		sv_numjobs;	/* number of job owned by server   */
 		int		sv_numque;	/* nuber of queues managed          */
-		long long	sv_jobidnumber;	/* next number to use in new jobid  */
-		time_t		sv_savetm;	/* time of server db update         */
+		long long sv_jobidnumber;	/* next number to use in new jobid  */
```

Comment:
Please put a tab between long long and sv_jobidnumber and others just to align it with the other data members.

Reply:
formatted to remove alignment